src/style.css

```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body * {
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+  color: rgba(61, 78, 97, 1);
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search-input {
+  background-color: white ;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  outline: none;
+  width: 100%;
+  border: 1px solid rgba(225, 231, 237, 1);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  transition: all 0.3s;
+  background-position: 26px 25px, 0 0;
+  text-indent: 62px;
+  height: 70px;
+  font-size: 16px;
+}
+
+.search-small {
+  background-position: 12px 14px, 0 0 ;
+  text-indent: 33px;
+  height: 42px;
+  font-size: 14px;
+  background-size: 11px, 100% ;
+}
```

Comment:
Rename it due to the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) p.2
![image](https://user-images.githubusercontent.com/80700247/225021038-f55dd03c-4139-4fd0-a9ea-9ca8976fbf43.png)